tests/upgrades/test_syncplan.py

```diff
@@ -54,60 +43,54 @@ def test_pre_sync_plan_migration(self):
         :expectedresults: Run sync plan create, get, assign and verify it should pass
 
         """
-        org = entities.Organization().create()
-        sync_plan_name = "Test_Sync_plan_Migration_{}".format(gen_string('alpha'))
-        sync_plan = entities.SyncPlan(organization=org, name=sync_plan_name).create()
-        product = entities.Product(organization=org).create()
-        entities.Repository(product=product).create()
+        org = entities.Organization(name=f'{request.node.name}_org').create()
+        sync_plan = entities.SyncPlan(
+            organization=org, name=f'{request.node.name}_syncplan', interval="hourly"
+        ).create()
+        product = entities.Product(organization=org, name=f'{request.node.name}_prod').create()
+        entities.Repository(product=product, name=f'{request.node.name}_repos').create()
         sync_plan.add_products(data={'product_ids': [product.id]})
         product.sync()
         product = product.read()
-        self.assertEqual(product.sync_plan.id, sync_plan.id)
-        sync_plan = sync_plan.read()
-        scenario_dict = {
-            self.__class__.__name__: {
-                'sync_plan_name': sync_plan_name,
-                'interval': sync_plan.interval,
-                'sync_date': sync_plan.sync_date,
-                'product_id': product.id,
-                'sync_plan_id': sync_plan.id,
-                'org_id': org.id,
-            }
-        }
-        create_dict(scenario_dict)
+        assert product.sync_plan.id == sync_plan.id
 
     @post_upgrade(depend_on=test_pre_sync_plan_migration)
-    def test_post_sync_plan_migration(self):
-        """Post-upgrade scenario that tests existing sync plans are working as
-        expected after satellite upgrade with migrating from pulp to katello
+    def test_post_sync_plan_migration(self, request, dependent_scenario_name):
+        """After upgrade, Sync interval update should work on existing sync plan(created before
+        upgrade)
 
         :id: badaeec2-d42f-41d5-bd85-4b23d6d5a724
 
         :steps:
             1. Verify sync plan exists and works as earlier
+            2. Check the all available sync_interval type update with pre-created sync_plan
```

Comment:
The check steps are really needed in steps. You can move them to expected results as example:
```
1. All available sync_intervals type should be updated in pre-created sync_plan.
```